src/main/java/org/zalando/nakadi/domain/Storage.java

```diff
@@ -70,6 +75,7 @@ public String toString() {
     }
 
     private String id;
+    @JsonProperty("storage_type")
```

Comment:
When @antban commented on this, the json sent by the user didn't have an ID (it was generated by nakadi as a random UUID). Now, they're pretty much identical, so the issue is less clear cut.